server/httpin/service.go

```diff
@@ -16,18 +16,19 @@ import (
 )
 
 type Service interface {
-	ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request)
+	ServeHTTP(c *plugin.Context, w http.ResponseWriter, req *http.Request)
 }
 
+// TODO <>/<>: 1/5: Can Handler be combined into Service?
```

Comment:
@mickmister @hanzei do you know if there were special considerations for why we split those 2? Seem easily combinable, just 1 struct?